sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -59,6 +59,7 @@ export interface Receiver<ReceivedMessageT> {
   /**
    * Returns a promise that resolves to a deferred message identified by the given `sequenceNumber`.
    * @param {Long} sequenceNumber The sequence number of the message that needs to be received.
+   * @param options - Options bag to pass an abort signal or tracing options.
```

Comment:
Do we actually call it a `bag`?

Reply:
We sure do in the guidelines :)

<img width="1021" alt="Screen Shot 2020-04-02 at 11 25 23 PM" src="https://user-images.githubusercontent.com/16890566/78330633-45089f00-7539-11ea-85e4-cc51cbf432ea.png">